app/inbound/calculation/process-calculation.js

```diff
@@ -18,7 +17,6 @@ const processCalculation = async (calculation) => {
       await savePlaceholderOrganisation({ sbi: calculation.sbi }, calculation.sbi)
       const savedCalculation = await saveCalculation(calculation, transaction)
       await saveFundings(calculation.fundings, savedCalculation.calculationId, transaction)
-      await updateCalculation(calculation.invoiceNumber, savedCalculation.calculationId, transaction)
```

Comment:
Removing as will never find match.  Cannot reliably match calculation to completed request due to splitting and transformation.